src/appservice.ts

```diff
@@ -17,7 +17,11 @@ export function getAppservice(config: BridgeConfig, registration: IAppserviceReg
         storage = new MemoryStorageProvider();
     }
 
-    const cryptoStorage = config.encryption?.storagePath ? new RustSdkAppserviceCryptoStorageProvider(config.encryption.storagePath) : undefined;
+    const cryptoStorage = config.encryption?.storagePath ?
+        new RustSdkAppserviceCryptoStorageProvider(
```

Comment:
should we loggy that we're setting up crypto (with the path)?

Reply:
Sure, couldn't hurt